credentials/credentials.go

```diff
@@ -86,6 +87,12 @@ type AuthInfo interface {
 	AuthType() string
 }
 
+var (
+	// ErrConnDispatched indicates the rawConn is dispatched by server credentials,
+	// and gRPC should keep rawConn open.
```

Comment:
ErrConnDispatched indicates that rawConn has been dispatched out of gRPC and the caller should not close rawConn.